lib/envar.ex

```diff
@@ -142,6 +142,34 @@ defmodule Envar do
     :ok
   end
 
+  @doc """
+  `require_env_file/1` load a file containing a line-separated list
+  of environment variables e.g: `.env`
+  Set the `value` of each environment variable.
+  Log an Error if the file is not available
+
+  ## Examples
+      iex> Envar.require_env_file(".env")
+      :ok
+
+      iex> Envar.require_env_file(".env_not_there")
+      :error
+
+  """
+  @spec require_env_file(binary) :: :ok
+  def require_env_file(filename) do
+    # check if the file exists:
+    path = Path.join(File.cwd!(), filename)
```

Comment:
Do we want to use `File.cwd!()` instead of `File.cwd()`
I'm not sure, I guess in this case we expect the [current directory](https://hexdocs.pm/elixir/1.12/File.html#cwd/0) to always exists

> In rare circumstances, this function can fail on Unix-like systems.

It there is an error we can let it crash

Reply:
I honestly don't know how I would test this ... but happy to change the invocation to `File.cwd()` ... 💭